package.json

```diff
@@ -12,22 +12,21 @@
     "Mihai Bazon <mihai.bazon@gmail.com> (http://lisperator.net/)"
   ],
   "repository": "mishoo/UglifyJS",
-  "main": "tools/node.js",
+  "main": "index.js",
   "bin": {
     "uglifyjs": "bin/uglifyjs"
   },
   "files": [
-    "bin",
-    "lib",
-    "tools",
+    "index.js",
     "LICENSE"
   ],
   "devDependencies": {
     "acorn": "~8.7.1",
     "semver": "~6.3.0"
   },
   "scripts": {
-    "test": "node test/compress.js && node test/mocha.js"
+    "test": "node test/compress.js && node test/mocha.js",
+    "prepack": "bin/uglifyjs.js --self --mangle --compress --output index.js && bin/uglifyjs.js bin/uglifyjs.js --mangle --compress --output bin/uglifyjs"
```

Comment:
I see what you did here, though I think we don't need to mangle/compress these because `npm` packages don't generally minify their source files &minus; in fact, to allow for more informative bug reports, we should probably do:
```
bin/uglifyjs --self --beautify --output index.js
```
so the code won't all end up on a single line.

Reply:
Would also include sourcemaps, so that when running node with [`--enable-source-maps`](https://nodejs.org/dist/latest-v19.x/docs/api/cli.html#--enable-source-maps), errors can point back to the original source

Reply:
sure, just add `--source-map` and it should produce `/index.map` accordingly :wink: